### DISCUSSION TITLE
manual config of projects

### DISCUSSION THREAD

**Discussion by gulsezim11** [DependencyTrack/dependency-track#2860] [Q&A]
### Current Behavior

I have configured the integration between DepTrack and DefectDojo. In the instruction, it is said that each project should be configured with project properties (defectdojo.engagementId). But I have about 1000 projects in deptrack, it is a lot of manual work to configure properties in each project. Are there any solution for this?

The instruction https://docs.dependencytrack.org/integrations/defectdojo/

### Proposed Behavior

Are there any ways to minimize manual work of configuration project properties? 

### Checklist

- [X] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#filing-issues)
- [X] I have checked the [existing issues](https://github.com/DependencyTrack/dependency-track/issues) for whether this enhancement was already requested

**Comment by valentijnscholten** [comment]
The properties can be set via the API.

**Comment by nscuro** [comment]
Indeed you'd have to script this kind of sync, and as @valentijnscholten said the DT API has endpoints for this.

**Comment by nscuro** [comment]
Also, please do not use issues to ask questions. I'll transfer this to discussions instead.

**Comment by gulsezim11** [comment]
Thanks to you all!
